components/styled/table/src/organisms/questionnaire-table-methods.ts

```diff
@@ -268,6 +268,21 @@ const createCellPropsForAnswer = (answer: QuestionnaireResponseItemAnswer): Cell
       text: answer.valueString,
     }
   }
+  if (answer.valueBoolean != null) {
```

Comment:
Could we just make sure these cases are covered in the tests? I think we should be able to cover this in the mapping unit tests by making sure the incoming object has some of these values

Reply:
<img width="611" alt="image" src="https://user-images.githubusercontent.com/63285990/217777344-0b55d43c-2561-45c7-a63c-27a98893bf74.png">
Are the ones I had in mind - should just be a case of making sure that summaryRecordsList contains some answers with valueInteger, valueDecimal etc. etc. and that the output is what you'd expect

Reply:
Yeah I'll get started on adding those in, thanks for pointing that out